src/main/java/com/bbaga/githubscheduledreminderapp/domain/sources/github/filters/AuthorFilter.java

```diff
@@ -18,12 +18,30 @@ public void configure(AbstractFilterConfig config) {
     @Override
     public Boolean filter(GitHubIssue issue) {
         List<String> includeAuthors = config.getIncludeAuthors();
+        List<String> excludeAuthors = config.getExcludeAuthors();
+
+        //Guard case - if both filters have nothing, filter nothing
+        if ((includeAuthors == null || includeAuthors.isEmpty()) &&
+            (excludeAuthors == null || excludeAuthors.isEmpty())) {
+            return false;
+        }
+
+        // Uses Include then exclude.
+        // If a name exists in both include and exclude, it will be excluded.
```

Comment:
Is this true? Looks to me the exclude list won't be checked if the include list isn't empty. Therefore if the login name is in the include list, it won't be excluded. So I think [this should be an assertTrue](https://github.com/bbaga/github-scheduled-reminder-app/pull/238/files#diff-ec3bcb6f4b40591027dd3602e539fc2555f5afb8ef6f5acf184d1661604671c0R112).

Reply:
I changed this after talking to you and missed updating the comment.